components/header/headerCompontents/NavbarCartIcon.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+import React from "react";
+import Link from "next/link";
+import CartDetails from "./CartDetails";
+import { useAppSelector } from "@/lib/hookts";
+import { RootState } from "@/lib/store";
+
+export default function NavbarCartIcon() {
+  const cartProducts = useAppSelector((state: RootState) => state.cartProducts);
+  const totalPrice = cartProducts.reduce(
+    (total, product) => total + product.price * product.quantity,
+    0
+  );
+  return (
+    <Link href="/cart">
+      <div className="relative lg:text-2xl bg-yellow-700  flex gap-2 px-6 py-2 rounded-md justify-center items-center cursor-pointer hover:bg-yellow-600">
+        <svg
```

Comment:
@Kamildeeal move all svgs to separate folder like `assets/icons` and just import it here and use it as component